repo/packages/K/kafka-zookeeper/0/package.json

```diff
@@ -11,7 +11,7 @@
     "zookeeper",
     "kafka"
   ],
-  "preInstallNotes": "Default configuration requires 3 agent nodes each with: 1.0 CPU | 4000 MB MEM | 5000 MB Disk\n\nBy using this service you agree to the license terms here: https://github.com/mesosphere/dcos-zookeeper/blob/master/LICENSE\n\nVersions: Apache ZooKeeper 3.4.11",
+  "preInstallNotes": "Default configuration requires 3 agent nodes each with: 1.0 CPU | 4000 MB MEM | 5000 MB Disk\n\nVersions: Apache ZooKeeper {{zookeeper-version}}",
```

Comment:
@rathohit311356 you have tested this? 
{{zookeeper-version}}

Reply:
please change it back to the version numbers as it was before.

Reply:
Changes Done